<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map_select</class>
 <widget class="QWidget" name="map_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>选择地图</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局背景 */
QWidget#map_select {
        background-color: #F5F7FA;
}

/* 顶部导航栏按钮 */
QPushButton[class=&quot;navBtn&quot;] {
        background-color: #FFFFFF;
        color: #4A5568;
        border: 1px solid #E2E8F0;
        border-radius: 6px;
        font-weight: bold;
        font-size: 12px;
}
QPushButton[class=&quot;navBtn&quot;]:hover {
        background-color: #EDF2F7;
        color: #2D3748;
}

/* 列表控件样式 */
QListWidget {
        background-color: transparent; /* 背景透明，显示底色 */
        border: none;                  /* 去掉丑陋的边框 */
        outline: none;                 /* 去掉选中时的虚线框 */
}

/* 底部操作面板 */
QFrame#bottomPanel {
        background-color: #FFFFFF;
        border-top-left-radius: 16px;
        border-top-right-radius: 16px;
        border: 1px solid #EDF0F5;
        border-bottom: none;
}

/* --- 按钮样式：主要操作 (Import) --- */
QPushButton[class=&quot;primaryBtn&quot;] {
        background-color: #4A90E2;
        color: white;
        border-radius: 8px;
        border: none;
        font-weight: bold;
        font-size: 14px;
}
QPushButton[class=&quot;primaryBtn&quot;]:hover {
        background-color: #5B9DF2;
}
QPushButton[class=&quot;primaryBtn&quot;]:pressed {
        background-color: #357ABD;
}

/* --- 按钮样式：次要操作 (Refresh) --- */
QPushButton[class=&quot;secondaryBtn&quot;] {
        background-color: #FFFFFF;
        color: #718096;
        border: 1px solid #E2E8F0;
        border-radius: 8px;
        font-weight: bold;
        font-size: 14px;
}
QPushButton[class=&quot;secondaryBtn&quot;]:hover {
        background-color: #F7FAFC;
        color: #4A90E2;
        border: 1px solid #4A90E2;
}

/* --- 滚动条美化 --- */
QScrollBar:vertical {
    border: none;
    background: transparent;
    width: 6px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: #CBD5E0;
    min-height: 20px;
    border-radius: 3px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>

   <item>
    <widget class="QWidget" name="topBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_Top">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbBack">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>← 返回</string>
        </property>
        <property name="class" stdset="0">
         <string>navBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_Title">
        <property name="text">
         <string>选择地图</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px; color: #2D3748; background: transparent; font-weight: bold;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="placeholder" native="true">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Content">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>

     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QFrame" name="bottomPanel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_Bottom">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbRefresh">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>刷新列表</string>
          </property>
          <property name="class" stdset="0">
           <string>secondaryBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbImport">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>+ 导入新地图</string>
          </property>
          <property name="class" stdset="0">
           <string>primaryBtn</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
